### ISSUE TITLE
patch and vcv rack

### ISSUE THREAD

**Issue by lorenaSalome** [dispersionlab/gitShow1#3]
@michaelpalumbo I am having problems opening the patch, vv rack shuts down everytime I try to penit, but when I try other vcv filesm they open without any problem. Let me know if you are having problems opening it. I am trying to figure it out. Thanks

**Comment by michaelpalumbo** [comment]
Hi Lorena, what happens if you open a version of it from an earlier commit?
On Jan 20, 2020, 12:44 AM -0500, Lorena Salomé <notifications@github.com>, wrote:
	@michaelpalumbo I am having problems opening the patch, vv rack shuts down everytime I try to penit, but when I try other vcv filesm they open without any problem. Let me know if you are having problems opening it. I am trying to figure it out. Thanks

**Comment by lorenaSalome** [comment]
@michaelpalumbo it is strange. I redid the patch, work on it, play and record something, close the patch and then tried to open it again and same problem. I think it is time to get a new computer :)

**Comment by michaelpalumbo** [comment]
Can you try posting something in the vcv rack forums? I wonder if there's something they can assist with?

Community.vcvrack.com
On Jan 20, 2020, 11:46 AM -0500, Lorena Salomé <notifications@github.com>, wrote:
	@michaelpalumbo it is strange. I redid the patch, work on it, play and record something, close the patch and then tried to open it again and same problem. I think it is time to get a new computer :)

**Comment by lorenaSalome** [comment]
@michaelpalumbo I found things online. I figured out the problem now. I am trying to fix it. The problem is one of the modules I am using, the Mverb. I will investigate more on it and keep you posted. I might have to not use the module

**Comment by lorenaSalome** [comment]
It was definitely the mverb module. I replaced it with another reverb. Can I commit and push the patch with the new reverb?

**Comment by michaelpalumbo** [comment]
Yes absolutely! So that's what was causing the crash?
On Jan 20, 2020, 8:44 PM -0500, Lorena Salomé <notifications@github.com>, wrote:
	It was definitely the mverb module. I replaced it with another reverb. Can I commit and push the patch with the new reverb?

**Comment by lorenaSalome** [comment]
yes, it was that! thank you. I just realized i didn't put the issue in the general one, sorry. 
I still have the saturation while I listen with a very simple patch.

**Comment by michaelpalumbo** [comment]
Hmm. the 'saturation' could be that rack's sample rate is different from
your sound card's sample rate? always good to check that when things sound
distorted. otherwise yes, I'd recommend taking this problem the vcv rack
forums as it could be something I can't help with :)
